packages/block-library/src/table/state.js

```diff
@@ -78,7 +78,11 @@ export function updateSelectedCell( state, selection, updateCell ) {
 		return state;
 	}
 
-	const tableSections = pick( state, [ 'head', 'body', 'foot' ] );
+	const tableSections = Object.fromEntries(
+		Object.entries( state ).filter( ( [ key ] ) =>
```

Comment:
Has a bunch of default attributes, no need for a fallback:

https://github.com/WordPress/gutenberg/blob/4f7c19e37d55bb6a419cc8eca94c634f1e71852e/packages/block-library/src/table/index.js#L24